### PR TITLE
Twitter auth

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

- add next-auth
- add twitter auth setup to .env.development
- add TwitterHandler.tsx
- setup theme for Roboto
- remove hello.ts from api
- update README
- check if users accounts is over 30 days old, has 1 tweet & 15 followers

## Notes
- will require some setup in Vercel for Prod - I've sent Marco a message to see if I can get access to CENNZnet twitter account to set up OAuth
- next-auth only works with Node versions <= 16

### CODE REVIEW

File: components/TwitterHandler.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+export const styles = {
+	username: css`
+		font-size: 14px;
+		position: absolute;
+		top: 8px;
+		right: 150px;
+		font-weight: bold;
+	`,
+	buttonContainer: css`
+		position: absolute;
```
Comment (eupston) [inline components/TwitterHandler.tsx:66]: could we put cursor pointer in here

Comment (aidan-starke) [reply components/TwitterHandler.tsx:66]: good point 😆

**Review by eupston — COMMENTED** [review]
looks good man, although doesn't seem to be authenticating me or give access back to the app? Also maybe we should have it pop up in a separate window? Or even as a modal if possible?
Screen Shot 2022-03-07 at 2 49 47 PM

**Comment by aidan-starke** [comment]
> looks good man, although doesn't seem to be authenticating me or give access back to the app? Also maybe we should have it pop up in a separate window? Or even as a modal if possible? Screen Shot 2022-03-07 at 2 49 47 PM

Dunno if I have that much control over it with next-auth but I'll look into it, this setup will currently only work with localhost:3000

**Review by eupston — APPROVED** [review]

**Pull request merged**